src/win32/findfile.c

```diff
@@ -68,6 +68,42 @@ static wchar_t* win32_walkpath(wchar_t *path, wchar_t *buf, size_t buflen)
 	return (path != base) ? path : NULL;
 }
 
+static int win32_find_git_for_windows_architecture_root(_findfile_path *root_path, const wchar_t *subdir)
+{
+	/* Git for Windows >= 2 comes with a special architecture root (mingw64 and mingw32)
+	 * under which "etc" and the "share" folders are located, check which we need (none is also ok) */
+
+	static const wchar_t *architecture_roots[3] = {
+		L"mingw64\\",
+		L"mingw32\\",
+		NULL,
+	};
```

Comment:
Using `ARRAY_SIZE` instead of a sentinel value would've been a bit easier to read, but that might be my personal taste, only.

Reply:
I tried to stick to the coding style in that file.